README.md

```diff
@@ -326,48 +330,51 @@ If you need to add some arguments as a parameters, use **success_args** and **fa
 def success_handler(param_1):
     print(f"param_1: {param_1}")
 
+
 def failure_handler(param_1, param_2):
     print(f"param_1: {param_1}")
     print(f"param_2: {param_2}")
 
 
 result = string_from_key(dictionary=user_info, key="first_name")
 
-result.handle(on_success=success_handler, 
-              on_failure=failure_handler,
-              success_args=1,
-              failure_args=(1, 2))
+result.handle(
+    on_success_handler=OnSuccessHandler(func=success_handler, args=(1,)),
+    on_failure_handler=OnFailureHandler(func=failure_handler, args=(1, 2))
+)
 ```
 
 ##### Additional parameters in combination with the Result itself
 
-Sometimes a handle function will need information about external parameters and also about the result itself. Now, is possible this combination thanks to `Result.__id__` identifier. 
+Sometimes a handle function will need information about external parameters and also about the result itself. Now, is possible this combination thanks to `Result.__id__` identifier.
 
 ```python
-    parameters = (1, Result.__id__, 2)
+args = (1, Result.__id__, 2)
 
-    def on_success(param_1: int, result: Result, param_2: int):
-        assert param_1 == 1
-        assert isinstance(result, Result)
-        assert result.value is True
-        assert param_2 == 2
 
-    def on_failure(param_1: int, result: Result, param_2: int):
-        assert param_1 == 1
-        assert isinstance(result, Result)
-        assert result.value == Error()
-        assert param_2 == 2
+def success_handler(param_1: int, result: Result, param_2: int):
+    assert param_1 == 1
+    assert isinstance(result, Result)
+    assert result.value is True
+    assert param_2 == 2
 
-    @meiga
-    def run():
-        result.handle(
-            on_success=on_success,
-            on_failure=on_failure,
-            success_args=parameters,
-            failure_args=parameters,
-        )
-
-    run()
+
+def failure_handler(param_1: int, result: Result, param_2: int):
+    assert param_1 == 1
+    assert isinstance(result, Result)
+    assert result.value == Error()
+    assert param_2 == 2
+
+
+@meiga
+def run():
+    result.handle(
+        n_success_handler=OnSuccessHandler(func=success_handler, args=args),
```

Comment:
typo: expects the `o` in parameter name `n_success_handler`

Reply:
Alright!